### DISCUSSION TITLE
Connect to Amazon RDS  SQL Server instance from Mirth Connect Database Writer

### DISCUSSION THREAD

**Discussion by snarayanan99** [nextgenhealthcare/connect#6352] [Q&A] [answered]
I need to connect to RDS Micrsoft SQL Server instance from my Mirth Connect application. Would the existing com.microsoft.sqlserver.jdbc.SQLServerDriver driver work against the Amazon RDS instance as well

**Comment by pacmano1** [comment, marked as answer]
The driver included in Mirth should work fine.
